src/bin/genemichaels.rs

```diff
@@ -82,14 +91,18 @@ impl FromStr for On {
     }
 }
 
-#[derive(Parser)]
+#[derive(Parser, Clone)]
 struct Args {
     #[arg(help = "Files to format in place; if none specified formats stdin and writes result to stdout")]
     files: Vec<PathBuf>,
-    #[arg(short, long, default_value_t = FormatConfig::default().max_width)]
-    line_length: usize,
     #[arg(short, long, help = "Won't emit any output")]
     quiet: bool,
+    #[arg(short, long, help = "Formats the entire project")]
+    project: bool,
+    #[arg(long, help = "Limits threads to specified count")]
```

Comment:
Would you mention that this is when using the above flag? Just so nobody gets confused adjusting this when formatting individual files.

Reply:
Good call